src/features/bivariate_color_manager/components/MiniLegend/MiniLegend.fixture.tsx

```diff
@@ -1,6 +1,5 @@
-import { useValue } from 'react-cosmos/fixture';
+import React from 'react';
```

Comment:
Are we really need react here?

Reply:
Idk why, but it was required and now it's now, so I've removed it.